synapse/tests/common.py

```diff
@@ -123,3 +123,21 @@ def __call__(self, *args, **kwargs):
 
     def wait(self, timeout=None):
         return self.event.wait(timeout=timeout)
+
+class SyncToAsyncCMgr():
```

Comment:
is this something that should live in 1. synapse.tests.utils, 2. synapse.coro or 3. synapse.common ?

Reply:
yes

Reply:
moved to tests.utils